server/pbench/bin/pbench-rsync-satellite

```diff
@@ -86,19 +86,16 @@ else
     dryrun=
 fi
 
+hosts="$(for host in $files;do echo ${host%%/*};done | sort -u )"
+
+tmp=$TMP/$PROG.$$
```

Comment:
This line needs to go:  tmp is set in the if-else above, so we don't want to reset it here. After this is done, the unit test
gold file for test-11 (and maybe test-10) will need updating to get the `tmp/pbench-rsync-satellite.XXXXX` form. That will allow the unit tests to pass.